tests/chart_tests/test_astronomer_config_syncer.py

```diff
@@ -278,3 +278,23 @@ def test_astronomer_config_syncer_disabled(self, kube_version):
         )
 
         assert len(docs) == 0
+
+    def test_astronomer_config_syncer_and_global_rbac_enabled_with_sa_create_disabled(self, kube_version):
+        """Test that config syncer service account, rbac  is disabled when config syncer and global rbac is enabled."""
+
+        # First rbacEnabled set to true and namespacePools disabled, should create a ClusterRole and ClusterRoleBinding
```

Comment:
This comment implies that there will be a "Second". Can we improve or remove this comment? Do we need to add another test?

Reply:
I unresolved this comment because I have not seen the answer to my questions and concerns.